typetag.go

```diff
@@ -149,3 +197,46 @@ func (xt *AccountAddressTag) MarshalBCS(bcs *Serializer) {
 func (xt *AccountAddressTag) UnmarshalBCS(bcs *Deserializer) {
 	xt.Value.UnmarshalBCS(bcs)
 }
+
+type StructTag struct {
+	Address    AccountAddress
+	Module     string // TODO: IdentifierWrapper ?
+	Name       string // TODO: IdentifierWrapper ?
```

Comment:
We don't need IdentifierWrapper.  That's an overoptimization on the Rust side.  A `box<str>` is one word less than a `String`